golfer/golfer.go

```diff
@@ -150,6 +158,10 @@ func GetInfo(db *sql.DB, name string) *GolferInfo {
 		panic(err)
 	}
 
+	if info.Discord.Valid {
+		info.DiscordUser, _ = discord.Session.User(info.Discord.String)
```

Comment:
does this make a network call? Not sure we want this in the happy path of every request if so